README.md

```diff
@@ -16,14 +16,6 @@ Or install it yourself as:
 
     $ gem install omniauth-linkedin-oauth2
 
-## Upgrading
-
```

Comment:
Instead of removing Upgrade section I would propose mentioning about changes in scopes

Reply:
The upgrade section is getting unwieldy I couldn't figure out how to word it anymore. I think it would make a lot more sense that this information goes in a the release notes or in a changelog instead of at the the top of the readme. Or if you have wording you think would be clear, we can do that too. I just was not sure where to go with this.